venues/ICLR.cc/2018/Conference/process/submissionProcess.js

```diff
@@ -5,8 +5,24 @@ function() {
     var CONF = 'ICLR.cc/2018/Conference';
     var PROGRAM_CHAIRS = CONF + '/Program_Chairs';
     var AREA_CHAIRS = CONF + '/Area_Chairs';
-    var BLIND_SUBMISSION = CONF + '/-/Blind_Submission';
+    var REVIEWERS = CONF + '/Reviewers';
     var AUTHORS = CONF + '/Authors';
+    var REVIEWERS_PLUS = REVIEWERS + '_and_Higher';
```

Comment:
What does it mean and_Higher? I'm not sure that this is the way to go.

Reply:
"reviewers plus" is a group consisting of reviewers, area chairs, and program chairs.
"area chairs plus" is just area chairs and program chairs.

we do it this way because the program chairs want people to be able to select a single "readers" option, and they want the options to be organized this way.

Reply:
I see, but the Reviewers group is not created by paper number, right? Same as plus groups. We only need to create once per all the conference. It shouldn't be in the process function, may be in the superuser-init?

Reply:
that's right, it's not dependent on paper number - but it's not being created in this process function. It's just a string variable that gets used later, in the "readers" field of the comment invitations. I found it easier to read this way.

Reply:
Ohhh, I'm sorry, discard my comment!